libraries/helpers.rb

```diff
@@ -0,0 +1,38 @@
+module Chocolatey
+  module Helpers
+    # Get the ChocolateyInstall directory from the environment.
+    def chocolatey_install
```

Comment:
@mwrock the new behavior tries the user env var first then falls back to fetching the Machine env var. This seems more robust than the previous helper method:

```ruby
# previous method
  def self.chocolatey_install
    ci_keys = ENV.keys.grep(/^ChocolateyInstall$/i)
    ci_keys.count > 0 ? ENV[ci_keys.first] : nil
  end
```

With my new method I was trying to cover the case where chocolatey has just been installed but the user ENV vars haven't been reloaded yet.